ui/app/templates/components/dashboard/vault-version-title.hbs

```diff
@@ -0,0 +1,9 @@
+<PageHeader as |p|>
+  <p.levelLeft>
+    <h1 class="title is-3" data-test-dashboard-version-header>
+      {{this.versionHeader}}
+      <Hds::Badge @text={{this.namespaceDisplay}} @icon="org" data-test-badge-namespace />
```

Comment:
We should hide this if it's the Community version (non-enterprise)

Reply:
That makes sense - is that because namespaces are an enterprise-only feature?